internal/build/docker.go

```diff
@@ -217,10 +212,11 @@ func (l *localDockerBuilder) buildFromDf(ctx context.Context, df Dockerfile, pat
 			Context:    archive,
 			Dockerfile: "Dockerfile",
 			Remove:     shouldRemoveImage(),
+			Version:    types.BuilderBuildKit,
```

Comment:
Here's where we tell it to use BuildKit.

Reply:
I think that's a good idea if it's easy, I'll try it first thing tomorrow morning.